WcaOnRails/app/helpers/notifications_helper.rb

```diff
@@ -68,13 +68,24 @@ def notifications_for_user(user)
     end
 
     user.delegated_competitions.visible.over
-        .includes(:delegate_report).where(delegate_reports: { posted_at: nil })
-        .each do |competition|
+      .each do |competition|
+        if !competition.delegate_report.posted_at && !competition.results_posted_at?
           notifications << {
+            text: "The delegate report and competition results for #{competition.name} have not been submitted.",
+            url: delegate_report_path(competition),
```

Comment:
I don't know that it makes sense to link to the delegate report here.... could you explain your reasoning? I feel like it should be fine to link to the competition, and we should make sure that competition page has useful warnings and links to take care of the delegate report and submitting results.

Reply:
After reviewing it seems even more useful to link to the results page instead. But I guess you are right, a delegate should get all necessary information from the competition's main page (therefore a message should show up which indicates the responsible delegates to submit the results).

Reply:
Does our main page show useful information for a delegate who has work to do? If not, could you file an issue with some ideas for improvements? =)

Reply:
I meant the competition main page, such as https://www.worldcubeassociation.org/competitions/MunichOpen2018
There is already a notification for missing delegate reports, so why not add another one for missing competition results?! :)